UI/CensusBrowseWnd.cpp

```diff
@@ -23,25 +23,25 @@ namespace {
     constexpr GG::Y ICON_BROWSE_ICON_HEIGHT(64);
 
     const GG::X BrowseTextWidth()
-    { return GG::X(FontBasedUpscale(200)); }
+    { return GG::X(FontBasedUpscale(230)); }
 }
 
 class CensusRowPanel : public GG::Control {
 public:
-    CensusRowPanel(GG::X w, GG::Y h, const std::string& name, double census_val, bool show_icon) :
+    CensusRowPanel(GG::X w, GG::Y h, const std::string& name, double census_val, int worlds, bool show_icon) :
         GG::Control(GG::X0, GG::Y0, w, h, GG::NO_WND_FLAGS),
-        m_show_icon(show_icon)
+        m_show_icon(show_icon),
+        m_worlds(nullptr)
```

Comment:
`shared_ptr` default initializes to `nullptr` or empty, so this is unnecessary

Reply:
done

Reply:
i was trying to add something along the lines of 'Please do not explicitly initialize pointers to nullptr since this is redundent' under #Pointers at https://www.freeorion.org/index.php?title=Code_Standards&action=edit but i do not have permission to do that
![schirmschuß](https://user-images.githubusercontent.com/37298400/149206026-9b8a0fd2-594c-4da2-8003-4bfb32941a16.png)